classes/locallib/config.php

```diff
@@ -72,6 +72,9 @@ public static function defaultvalues() {
             'recordingstatus_enabled' => false,
             'meetingevents_enabled' => false,
             'participant_moderator_default' => '0',
+            'participant_guestlink_enabled' => '1',
+            'participant_guestlink_default' => '0',
+            'custom_logo' => '',
```

Comment:
This is unrelated to this PR

Reply:
The logo is for the guest access page. We did everything exactly like in greenlight. More on that above.